src/Abp/Application/Features/FeatureCheckerExtensions.cs

```diff
@@ -11,14 +11,14 @@ namespace Abp.Application.Features
     /// Some extension methods for <see cref="IFeatureChecker"/>.
     /// </summary>
     public static class FeatureCheckerExtensions
-    {       
+    {
         /// <summary>
-        /// Gets value of a feature by it's name. This is sync version of <see cref="IFeatureChecker.GetValueAsync(string)"/>
+        /// Gets the value of a feature by its name. This is the sync version of <see cref="IFeatureChecker.GetValueAsync(string)"/>
         /// 
-        /// This is a shortcut for <see cref="GetValue(IFeatureChecker, int, string)"/> that uses <see cref="IAbpSession.TenantId"/> as tenantId.
-        /// So, this method should be used only if TenantId can be obtained from the session.
+        /// This is a shortcut for <see cref="GetValue(IFeatureChecker, int, string)"/> that uses <see cref="IAbpSession.TenantId"/>.
+        /// Note: This method should only be used if the TenantId can be obtained from the session.
         /// </summary>
-        /// <param name="featureChecker"><see cref="IFeatureChecker"/> instance</param>
+        /// <param name="featureChecker"><see cref="IFeatureChecker"/>instance</param>
```

Comment:
This space shouldn't be removed. It's meant to be rendered as `IFeatureChecker instance`.

There are multiple instances of this.